spot_rl_experiments/experiments/skill_test/test_robot_view_poses.py

```diff
@@ -0,0 +1,240 @@
+import os
+import os.path as osp
+
+import numpy as np
+from perception_and_utils.utils.generic_utils import map_user_input_to_boolean
+from spot_rl.utils.retrieve_robot_poses_from_cg import get_view_poses
+from spot_rl.utils.waypoint_estimation_based_on_robot_poses_from_cg import (
+    get_navigation_points,
+)
+
+# NYC lab
+bboxs_info_nyc = {
```

Comment:
do we want to make these json files that we load from?

Reply:
These are isolated/mined receptacle information from CG.json, thus they will keep changing as we switch CGs as well as their format with standard waypoints.yaml differs thus we have hard coded these temporary values.